bitnami/jenkins/templates/init-hook-scripts-configmap.yaml

```diff
@@ -0,0 +1,21 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.initHookScripts }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "common.names.fullname" . }}-init-hook-scripts
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+data:
+  {{- range $filename, $content := .Values.initHookScripts }}
+  {{ $filename }}: |
+    {{- $content | nindent 4 }}
```

Comment:
Should we support templates here and use `common.tplvalues.render` instead?

Reply:
I'm not very good at English, and I've just barely moved beyond the beginner level with Helm charts, so I don't understand what you're talking about.